aws/test/test-cluster/kops-cluster.tf

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
If you change to keep the networking (and you have to actually) the test cluster will not work as is (and that is why it was removed)